spyder/app/tour.py

```diff
@@ -144,68 +144,74 @@ def get_tour(index):
+                           "evaluating it. You can get automatic "
+                           "completions while typing by pressing the "
+                           "<b>Tab</b> key, or automatically by enabling "
+                           "on-the-fly completions in Spyder's Preferences. "
+                           "<br><br>The Editor comes "
+                           "with a line number area (highlighted here in red) "
+                           "where Spyder shows warnings and syntax errors. "
+                           "They can help you to detect potential problems "
+                           "before running your code.<br><br>"
                            "You can also set debug breakpoints in the line "
-                           "number area, by doing a double click next to "
-                           "a non-empty line."),
+                           "number area by clicking next to a "
+                           "any non-empty line."),
               'widgets': [sw.editor],
               'decoration': [sw.editor_line_number_area]},
 
-             {'title': _("The IPython console"),
-              'content': _("This is one of panes where you can run or "
-                           "execute the code you wrote on the Editor. To do it "
-                           "you need to press the <b>F5</b> key.<br><br>"
-                           "This console comes with several "
+             {'title': _("IPython Console"),
+              'content': _("This is where you can run Python code, either "
+                           "from the Editor or interactively. To run the "
+                           "current file, press <b>F5</b> by default, "
+                           "or press <b>Ctrl+Enter</b> "
+                           "(<b>Cmd-Return</b> on Mac) to execute the current "
+                           "line or selection.<br><br>"
+                           "This console comes with many  "
                            "useful features that greatly improve your "
-                           "programming workflow (like syntax highlighting and "
-                           "inline plots). If you want to know more about them, "
-                           "please follow this <a href=\"{qtconsole_link}\">link</a>.<br><br>"
-                           "{button_text}").format(
-                           qtconsole_link=qtconsole_link, button_text=button_text),
+                           "programming workflow, like syntax highlighting, "
+                           "autocompletion, plotting and 'magic' commands. "
+                           "To learn more, check out the "
+                           f"<a href=\"{qtconsole_link}\">documentation</a>."
+                           f"<br><br>{button_text}"),
               'widgets': [sw.ipython_console],
-              'run': ["li = list(range(100))", "d = {'a': 1, 'b': 2}"]
+              'run': [
+                  "test_list_tour = list(range(100))",

```

Comment:
```suggestion
                  "test_list_tour = [1, 2, 3, 4, 5]",
```

Just to make this code less scary to newbies (but I like the new names for these variables).

Reply:
Sure, makes more sense.

Glad you like the names. I changed them primarily to avoid overwriting any variables in the user's current namespace, to minimize the tour's side effects.